packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -12,7 +12,8 @@
         "packageId": "Amazon.CDK.Alexa.Ask",
         "signAssembly": true,
         "assemblyOriginatorKeyFile": "../../key.snk",
-        "versionSuffix": "-devpreview"
+        "versionSuffix": "-devpreview",
+        "iconUrl": "https://raw.githubusercontent.com/aws/aws-cdk/master/logo/png/AWS-Cloud-Development-Kit_Icon_64_Squid.png"
```

Comment:
Maybe we should create a copy of this file in some other location in the repo (e.g. `logo/nuget-package-icon.png`).

Reply:
I was thinking the same & am glad you're bringing this up!

How about `logo/default-dark-64.png`, so that's a direct/simple way to get a "dark" version of the logo in "64px" resolution. You onboard with that?

Reply:
It should be evident that this is consumed by the package managers so people don’t take an initiative and rename this thinking it’s just some files sitting around.